.github/scripts/check_android_so.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+ERROR_COUNT=0
+
+function error() {
+    ((ERROR_COUNT++))
+    echo "$@" 1>&2
+}
+
+function warn() {
+    echo "$@" 1>&2
+}
+
+function check_so() {
+    SO_FILE="$1"
+    local OLD_ERROR_COUNT="$ERROR_COUNT"
+
+    if [ ! -f "$SO_FILE" ]; then
+        error "USAGE: $0 <SO_FILE_OR_DIR>"
+        error "Shared object must be a file."
+        return
+    fi
+
+    SCAN_RESULT="$(scanelf -qT "$SO_FILE")"
+
+    if [ "$SCAN_RESULT" != "" ]; then
+        error "RELOCATIONS in: '$SO_FILE'"
+        warn "$SCAN_RESULT"
+    fi
```

Comment:
This is the relevant check.

The other check blow is still relevant but less pressing.